public/apps/configuration/panels/audit-logging/audit-logging-edit-settings.tsx

```diff
@@ -121,9 +133,13 @@ export function AuditLoggingEditSettings(props: AuditLoggingEditSettingProps) {
       };
 
       if (props.setting === 'general') {
-        addSuccessToast('General settings saved');
+        addSuccessToast(
```

Comment:
Nit: Should this say "general settings save _for_ ..."

Reply:
This is actually added on as part of the helper function, which evaluates to:
```
for ${cluster.label || 'Local cluster'}
```
This is to prevent the toasts from saying "General settings saved for" when the multiple datasource feature is not enabled.